### PR TITLE
change dependency vscode-nsfw to Axosoft/nsfw

### LINKED ISSUE #4105: change dependency vscode-nsfw to Axosoft/nsfw
vscode-nsfw (https://github.com/Tyriar/vscode-nsfw/commits/master) is more than one year old.
And if we compare it with its origin Axosoft/nsfw (https://github.com/Axosoft/nsfw/commits/master), vscode-nsfw does not have extra enhancements / bug fixes while it is 20 commits behind as of today.

Maybe we should use Axosoft/nsfw instead.

### PR DESCRIPTION
**Pull request by elaihau**
- resolves #4105

Signed-off-by: elaihau <liang.huang@ericsson.com>

### CODE REVIEW

File: packages/core/package.json [hunk 1]
```diff
@@ -30,6 +30,7 @@
     "inversify": "^4.14.0",
     "lodash.debounce": "^4.0.8",
     "lodash.throttle": "^4.1.1",
+    "nsfw": "^1.2.0",
```
Comment (marcdumais-work) [inline packages/core/package.json]: We used to use this version of nsfw, before adopting the one from vscode. I'll check if this version is already covered by a CQ or not.

Comment (marcdumais-work) [reply packages/core/package.json]: > I'll check if this version is already covered by a CQ or not.

It's not - we had approval for `vscode-nsfw`. We'll need a new CQ.

Comment (elaihau) [reply packages/core/package.json]: Do we need a CQ even for a dependency version change?

Comment (benoitf) [reply packages/core/package.json]: yes for dependencies. For build and test dependencies we can request CQ with ranges ( like 1.2.0 and higher)

**Comment by marcdumais-work** [comment]
The virtual memory leak in nsfw has apparently been fixed: https://github.com/Axosoft/nsfw/issues/48

Depending who is fastest to release a version that has this fix, we might wait a little before switching the nsfw we use. This way, if Microsoft are quicker, we could continue using `vscode-nsfw` and have an up-to-date nsfw faster.

**Comment by elaihau** [comment]
> The virtual memory leak in nsfw has apparently been fixed: [Axosoft/nsfw#48](https://github.com/Axosoft/nsfw/issues/48)
> 
> Depending who is fastest to release a version that has this fix, we might wait a little before switching the nsfw we use. This way, if Microsoft are quicker, we could continue using `vscode-nsfw` and have an up-to-date nsfw faster.

4 days ago AxSoft/nsfw had a 1.2.1 release that contains the memory leak fix: https://github.com/Axosoft/nsfw/commits/master

I will update this PR to use the latest.

**Comment by akosyakov** [comment]
@marcdumais-work I wonder in this case is it possible to make an exception as well and merge?

**Comment by akosyakov** [comment]
@elaihau please rebase and merge, that's an important fix

**Comment by elaihau** [comment]
@akosyakov rebased and build passed. could you please review and approve?

**Comment by marcdumais-work** [comment]
@elaihau LGTM, but I'll briefly test before approving the PR.

**Comment by marcdumais-work** [comment]
I quickly tested and it seems to work well. I also confirmed that the [virtual memory leak](https://github.com/theia-ide/theia/issues/1284) that used to be present, is not there any more.

**Pull request merged**